CHANGES.rst

```diff
@@ -0,0 +1,13 @@
+Changes
+=======
+
+This is a record of all past ``gettsim`` releases and what went into them in reverse
+chronological order. We follow `semantic versioning <https://semver.org/>`_ and all
+releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_.
```

Comment:
Starting at 0.2? Happy to delete 0.1, too.

Reply:
I kept it for completeness and to point out Eric's contribution separately.